### ISSUE TITLE
There is a bug in declare abstract class

### ISSUE THREAD

**Issue by doctorcode-co** [Swatinem/rollup-plugin-dts#43]
This is my source definition file:
```
import MemberTypes from "./memberTypes";
import TypeInfo from "./typeInfo";
export default abstract class MemberInfo {
    abstract readonly name: string;
    abstract readonly declaringType: TypeInfo;
    abstract readonly memberType: MemberTypes;
}
```
after bundle with other types result is:
```
default abstract class MemberInfo {
    abstract readonly name: string;
    abstract readonly declaringType: TypeInfo;
    abstract readonly memberType: MemberTypes;
}
```
Instead, **"default abstract"** should be **"declare abstract"** output

**Comment by Swatinem** [comment]
Thanks for the report. I completely overhauled how this is handled and published `v1.1.4`, which also includes a proper fix for #42

**Comment by doctorcode-co** [comment]
Thanks for speed response. The problems mentioned are solved but a new error is seen that _was not seen in version 1.1.2._
This is our def source:
```
import { MethodParametersToken } from './metadataTokens';
export default abstract class MethodBase extends MemberInfo {
    method: Function & {
        [MethodParametersToken]?: ParameterInfo[];
    };
    constructor(method: Function);
    readonly isConstructor: boolean;
    getParameters(): ParameterInfo[];
    invoke(obj: any, parameters: any[]): any;
}
```
The result is:
```
declare abstract class MethodBase extends MemberInfo {
    method: Function & {
        [MethodParametersToken]?: ParameterInfo[];
    };
    constructor(method: Function);
    readonly isConstructor: boolean;
    getParameters(): ParameterInfo[];
    invoke(obj: any, parameters: any[]): any;
}
```
The bug is in  **[MethodParametersToken]?: ParameterInfo[];** because it is not imported.

**Comment by Swatinem** [comment]
Sorry for the delay, I fixed this in `1.1.5`. Please create a new issue next time :-)
